weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -90,89 +85,132 @@ def should_skip(self):
 
 class GribSplitter(FileSplitter):
 
+    def __init__(self, input_path: str, output_info: OutFileInfo,
+                 force_split: bool = False, logging_level: int = logging.INFO):
+        super().__init__(input_path, output_info,
+                         force_split, logging_level)
+        self.output_info.set_formatting_if_needed('_{typeOfLevel}_{shortName}')
+        self.split_dims = self._get_split_dims()
+
     def split_data(self) -> None:
+        if not self.split_dims:
+            raise ValueError('No splitting specified in template.')
         outputs = dict()
```

Comment:
Optional nit: Move this line after the `should_skip` if block, which returns early.